features/column-fill.yml.dist

```diff
@@ -0,0 +1,47 @@
+# Generated from: column-fill.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: high
+  baseline_low_date: 2017-03-07
+  baseline_high_date: 2019-09-07
+  support:
+    chrome: "50"
+    chrome_android: "50"
+    edge: "12"
+    firefox: "52"
+    firefox_android: "52"
+    safari: "9"
+    safari_ios: "9"
+compat_features:
+  # ⬇️ Same status as overall feature ⬇️
+  # baseline: high
+  # baseline_low_date: 2017-03-07
+  # baseline_high_date: 2019-09-07
+  # support:
+  #   chrome: "50"
+  #   chrome_android: "50"
+  #   edge: "12"
+  #   firefox: "52"
+  #   firefox_android: "52"
+  #   safari: "9"
+  #   safari_ios: "9"
+  - css.properties.column-fill
+
+  # baseline: high
+  # baseline_low_date: 2020-07-28
+  # baseline_high_date: 2023-01-28
+  # support:
+  #   chrome: "50"
+  #   chrome_android: "50"
+  #   edge: "12"
+  #   firefox: ≤72
+  #   firefox_android: "79"
+  #   safari: "9"
+  #   safari_ios: "9"
+  - css.properties.column-fill.auto
+  - css.properties.column-fill.balance
```

Comment:
The Firefox data is dubious. I sent https://github.com/mdn/browser-compat-data/pull/24585 to fix it.

Reply:
This is now merged in BCD. We'll pick this up in web-features on the next BCD release and upgrade.